convert_old_alw_to_new_structure.py

```diff
@@ -0,0 +1,26 @@
+import csv
+#csv file from which the original data is read
+with open('hr.contract (8).csv') as csvfile1: 
```

Comment:
Do not hard code the file, create a variable at the top of the file that stores the filename... also please provide descriptive variable names... e.g. instead of csvfile1 it can be called csvsourcefile